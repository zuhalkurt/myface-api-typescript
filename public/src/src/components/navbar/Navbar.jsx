import React from "react";
import {  Link } from "react-router-dom";

export function Navbar () {
    return <nav>
           <Link to="posts">Posts</Link>
           <Link to="users">Users</Link>
           <Link to="posts/create">Create Post</Link>
           <Link to="users/create">Create User</Link>
        </nav>

}
