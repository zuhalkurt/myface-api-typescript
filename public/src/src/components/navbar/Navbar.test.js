import React from 'react';
import { render, screen} from '@testing-library/react';
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import { Navbar } from './Navbar';

test("should match snapshot", async () => {
    
    const tree = renderer.create(<BrowserRouter> <Navbar/> </BrowserRouter>).toJSON();

    expect(tree).toMatchSnapshot();})