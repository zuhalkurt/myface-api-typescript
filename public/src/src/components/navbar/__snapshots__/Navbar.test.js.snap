// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Array [
  " ",
  <nav>
    <a
      href="/posts"
      onClick={[Function]}
    >
      Posts
    </a>
    <a
      href="/users"
      onClick={[Function]}
    >
      Users
    </a>
  </nav>,
  " ",
]
`;
